test/parse/TestEnumParser.cpp

```diff
@@ -332,6 +332,7 @@ BOOST_AUTO_TEST_CASE(UnlockableItemTypeParser)
     CHECK_ENUM_AND_RESULT("ShipHull", UIT_SHIP_HULL, UnlockableItemType, parse::unlockable_item_enum_grammar);
     CHECK_ENUM_AND_RESULT("ShipDesign", UIT_SHIP_DESIGN, UnlockableItemType, parse::unlockable_item_enum_grammar);
     CHECK_ENUM_AND_RESULT("Tech", UIT_TECH, UnlockableItemType, parse::unlockable_item_enum_grammar);
+    CHECK_ENUM_AND_RESULT("Policy", UIT_POLICY, UnlockableItemType, parse::unlockable_item_enum_grammar);
```

Comment:
@Cjkjvfnby @adrianbroher any hints why this wasn't enough to get the CI test to pass? I'm getting
> /freeorion/test/parse/TestEnumParser.cpp:328: error: fatal error in "UnlockableItemTypeParser": Untested enumeration value.

Reply:
Looks to me like you need to modify that line you're getting the error from, to specify there are now 6 unlock types:

```
BOOST_REQUIRE_MESSAGE(NUM_UNLOCKABLE_ITEM_TYPES == 5, "Untested enumeration value.");```